xarray/core/indexing.py

```diff
@@ -178,8 +178,9 @@ def convert_label_indexer(index, label, index_name="", method=None, tolerance=No
             else _asarray_tuplesafe(label)
         )
         if label.ndim == 0:
+            label_value = label[()] if label.dtype.kind in "mM" else label.item()
```

Comment:
```suggestion
            # see https://github.com/pydata/xarray/pull/4292/files for details
            label_value = label[()] if label.dtype.kind in "mM" else label.item()
```

Reply:
is there a reason why you point to the changes of this PR? If not, I would use
```suggestion
            # see https://github.com/pydata/xarray/pull/4292 for details
            label_value = label[()] if label.dtype.kind in "mM" else label.item()
```

Reply:
Agree. Will commit yours for the sake of expediancy